packages/paste-core/components/timeline/package.json

```diff
@@ -36,7 +36,9 @@
     "@twilio-paste/detail-text": "^3.1.0",
     "@twilio-paste/disclosure-primitive": "^2.1.1",
     "@twilio-paste/icons": "^12.0.0",
+    "@twilio-paste/paragraph": "^10.1.1",
     "@twilio-paste/reakit-library": "^2.0.0",
+    "@twilio-paste/skeleton-loader": "^6.1.0",
```

Comment:
question: was skeleton loader added for use in the story file? I know eslint throws an error, but we don't add dependencies for use in the stories, just for use in the actual components we're exporting from the package. if it's annoying to see the red lines you can ignore the extraneous dependencies rule for the story file